docs/source/app-dev/services.rst

```diff
@@ -159,6 +159,19 @@ Verbosity
 
 See :ref:`verbosity` above.
 
+.. note::
+
+  The RPCs exposed as part of the transaction and active contracts services make use of offsets.
+
+  An offset is an ever-growing number assigned by the participant to each transaction as they are received from the ledger.
+  This number is opaque to the client and must be used as-is (i.e. cliend-side transformations like incrementing it are not guaranteed to produce a valid offset).
```

Comment:
```suggestion
  This number is opaque to the client and must be used as-is (i.e. client-side transformations like incrementing it are not guaranteed to produce a valid offset).
```

Reply:
I believe we guarantee that they can be lexicographically ordered. Worth mentioning?

Reply:
Mmmm, we do and we mention it in the (linked) referenced docs where they are just an array of byte. Here I call them just "ever-growing number"s for simplicity. Saying that they can be lexicographically ordered seems redundant. Does that make sense?

Reply:
I think ever-growing number is somewhat misleading. You basically give the impression of something with very strong guarantees (e.g., consecutive) and then step back in the next sentence. I personally find it more intuitive to go with "they’re completely opaque, you can only order them lexicographically” but both work so your choice.

Reply:
Better? https://github.com/digital-asset/daml/pull/10180/commits/e857a7cf3f62c30816b79de67ea0dcb689a125a3

Reply:
I'll assume yes, let me know if you prefer me to change the wording.